### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by Sameony**
added Login and signup pages
image
image

### CODE REVIEW

File: Scripts/loginSignup.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const container = document.querySelector(".formContainer");
+const switchButtons = document.querySelectorAll(".pointer");
+
+switchButtons.forEach(element => {
+    element.addEventListener("click",() => {
+        container.classList.toggle("sign-in");
+        container.classList.toggle("sign-up");
+    });
+    console.log("da")
```
Comment (vaibhav-mahalle) [inline Scripts/loginSignup.js:9]: is there any reason for console.log here ?

File: Templates/loginSignup.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                    </h1>
+                </div>
+                <div class="img sign-in">
+                    <img src="/Assets/Images/kyotHugCat.jpg" alt="">
+                </div>
+            </div>
+            <!-- Sign up -->
+            <div class="half-wrapper align-items-center flex-col">
+                <div class="img sign-up">
+                    <img src="/Assets/Images/kyotComfy.jpg" alt="">
```
Comment (tanishq20) [inline Templates/loginSignup.html:119]: `alt` is missing in `img` tag

**Review by tanishq20 — APPROVED** [review]
Very clean code.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes. please look into it.

File: Scripts/loginSignup.js (hunk 1 above)
Comment (Sameony) [reply Scripts/loginSignup.js:9]: oh, Thanks for pointing out 
i missed it

File: Templates/loginSignup.html (hunk 2 above)
Comment (Sameony) [reply Templates/loginSignup.html:119]: oh, Thanks for pointing out

**Pull request merged**